src/Extensions/SiteTreeContentReview.php

```diff
@@ -160,28 +160,15 @@ public static function merge_owners(SS_List $groups, SS_List $members)
      */
     public function updateCMSActions(FieldList $actions)
     {
-        if ($this->canBeReviewedBy(Security::getCurrentUser())) {
-            Requirements::css('silverstripe/contentreview:client/dist/styles/contentreview.css');
-            Requirements::javascript('silverstripe/contentreview:client/dist/js/contentreview.js');
-
-            $reviewTitle = LiteralField::create(
-                "ReviewContentNotesLabel",
-                "<label class=\"left\" for=\"Form_EditForm_ReviewNotes\">" . _t("ContentReview.CONTENTREVIEW", "Content due for review") . "</label>"
-            );
-
-            $ReviewNotes = LiteralField::create("ReviewNotes", "<textarea class=\"no-change-track\" id=\"Form_EditForm_ReviewNotes\" name=\"ReviewNotes\" placeholder=\"" . _t("ContentReview.COMMENTS", "(optional) Add comments...") . "\" class=\"text\"></textarea>");
-
-            $quickReviewAction = FormAction::create("savereview", _t("ContentReview.MARKREVIEWED", "Mark as reviewed"))
-                ->addExtraClass('btn btn-primary');
-
-            $allFields = CompositeField::create($reviewTitle, $ReviewNotes, $quickReviewAction)
-                ->addExtraClass('review-notes field');
+        if (!$this->canBeReviewedBy(Security::getCurrentUser())) {
+            return;
+        }
 
-            $reviewTab = Tab::create('ReviewContent', $allFields);
-            $reviewTab->addExtraClass('contentreview-tab');
+        Requirements::css('silverstripe/contentreview:client/dist/styles/contentreview.css');
+        Requirements::javascript('silverstripe/contentreview:client/dist/js/contentreview.js');
```

Comment:
Probably not relevant now, but would be worth doing to future proof yourself:

Use ModuleLoader so that this module could potentially move to the vendor folder easily:
```
        $module = ModuleLoader::getModule('silverstripe/asset-admin');
        Requirements::javascript($module->getRelativeResourcePath("client/dist/js/bundle.js"));
```